File: app/src/main/java/com/kleber/beautyplanner/Agenda/AgendaActivity.java
package com.kleber.beautyplanner.Agenda;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.kleber.beautyplanner.R;

public class AgendaActivity extends AppCompatActivity {
    CustomCalendarView customCalendarView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_agenda);

        customCalendarView = (CustomCalendarView)findViewById(R.id.custom_calendar_view);
    }
}

File: app/src/main/java/com/kleber/beautyplanner/Agendas/AgendaActivity.kt
package com.kleber.beautyplanner.Agendas

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.kleber.beautyplanner.R

class AgendaActivity : AppCompatActivity() {
    var customCalendarView: CustomCalendarView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_agenda)
        customCalendarView = findViewById<View>(R.id.custom_calendar_view) as CustomCalendarView
    }
}